src/scss/pages/pages.scss

```diff
@@ -67,3 +67,9 @@
     }
   }
 }
+
+ // TODO: remove duplicate styles when workspace.scss imports all styles 
+.ordino-info-text {
+  color: map-get($theme-colors, "ordino-gray-4");
+  font-size: 1.2em;
+}
```

Comment:
@oltionchampari I adressed this TODO with the following PRs:

* https://github.com/phovea/phovea_ui/pull/141
* https://github.com/Caleydo/ordino/pull/276
* https://github.com/Caleydo/ordino_public/pull/86

In general, we should avoid setting font-size and colors using custom CSS classes and use the BS utility classes instead.